.github/workflows/trivy-scan.yml

```diff
@@ -3,31 +3,11 @@ name: Trivy
 on:
   workflow_dispatch:
   schedule:
-    - cron: '30 20 * * *'
+    - cron: "30 20 * * *"
 
 jobs:
-  ubuntu-build:
-    name: Build on Ubuntu
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: Set up JDK 17
-        uses: actions/setup-java@v3
-        with:
-          distribution: 'temurin'
-          java-version: 17.0.7
-      - name: Build with Gradle
-        env:
-          packageUser: ${{ github.actor }}
-          packagePAT: ${{ secrets.GITHUB_TOKEN }}
-        run: ./gradlew build -x check -x test -x :mssql-examples:build
-      - name: Create lib directory if not exists
-        run: mkdir -p ballerina/lib
-      - name: Run Trivy vulnerability scanner
-        uses: aquasecurity/trivy-action@master
-        with:
-          scan-type: 'rootfs'
-          scan-ref: '/github/workspace/ballerina/lib'
-          format: 'table'
-          timeout: '10m0s'
-          exit-code: '1'
+  call_workflow:
+    name: Run Trivy Scan Workflow
+    if: ${{ github.repository_owner == 'ballerina-platform' }}
+    uses: ballerina-platform/ballerina-standard-library/.github/workflows/trivy-scan-template.yml@main
+    secrets: inherit
```

Comment:
don't we have to skip this `-x :mssql-examples:build`?

Reply:
Yes we have to introduce the flag.

Reply:
Fixed in [ecdecee](https://github.com/ballerina-platform/module-ballerinax-mssql/pull/757/commits/ecdecee577577cfea464b800da39bde975c04d4b)